current_fall2025/scripts/app.py

```
@@ -30,170 +80,190 @@ def load_llm():
-        st.session_state.db_conn.autocommit = True
+        try:
+            st.session_state.db_conn = psycopg2.connect(
+                host=os.getenv("PGHOST"),
+                port=os.getenv("PGPORT"),
+                database=os.getenv("PGDATABASE"),
+                user=os.getenv("PGUSER"),
+                password=os.getenv("PGPASSWORD"),
+                sslmode=os.getenv("PGSSLMODE", "prefer"),
+            )
+            st.session_state.db_conn.autocommit = True
+        except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)